Sources/Corvus/Authentication/CorvusUser.swift

```diff
@@ -15,14 +15,9 @@ public final class CorvusUser: CorvusModel {
     @Field(key: "name")
     public var name: String
```

Comment:
What if we name this `username`? It might just be confused with the human name, e.g. first and last name, of a human user? We might argue that CorvusUser.username is a bit redundant but I think that the concept of a username is pretty known. Alternative might be just `userId`?